src/main/java/at/hannibal2/skyhanni/features/rift/everywhere/Ubikreminder.kt

```diff
@@ -0,0 +1,37 @@
+package at.hannibal2.skyhanni.features.rift.everywhere
+
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.features.rift.RiftAPI
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.hours
+
+@SkyHanniModule
+object Ubikreminder {
+
+    private val config get() = RiftAPI.config.ubikReminder
+
+    private var isTimerRunning = false
+    private val messageRegex = Regex("ROUND [1-9] \\(FINAL\\):")
+
+
+    @SubscribeEvent
+    fun onChat(event: LorenzChatEvent) {
+        if (!RiftAPI.config.ubikReminder) return
```

Comment:
before checking if the feautre is enabled, pleas add additional checks, e.g. if the player is in the rift, the right region of the rift, etc

Reply:
you would probably also want a reminder of this outside the rift, so at most this should be a setting imo

Reply:
Well the timer would only start inside of the rift, no?

Reply:
yeah, i probably didn't notice what hannibal was replying to